### PR TITLE
WIP: main/nfs-utils: do not run statd as root

### PR DESCRIPTION
**Pull request by HRio**

### CODE REVIEW

File: main/nfs-utils/nfs-utils.post-upgrade [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+ver_new="$1"
+ver_old="$2"
+
+if [ "$(apk version -t "$ver_old" '2.3.2-r2')" = '<' ]; then
+	# migrate owner from root: -> statd:
+	chown statd: /var/lib/nfs \
```
Comment (HRio) [inline main/nfs-utils/nfs-utils.post-upgrade]: @ncopa how to handle that rpc.statd should not be running when doing this? are we "allowed" to restart processes during upgrade?

Comment (nmeum) [reply main/nfs-utils/nfs-utils.post-upgrade]: Generally speaking I would *personally* prefer avoiding automigration magic in post install scripts and instead print a message informing the user of the necessary migration steps.

Comment (tcely) [reply main/nfs-utils/nfs-utils.post-upgrade]: Given that the old process is running as root, does this chown affect it at all?

File: main/nfs-utils/nfs-utils.pre-install [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+adduser -S -D -h /var/lib/nfs -s /sbin/nologin -G nogroup -g statd statd 2>/dev/null
```
Comment (tcely) [inline main/nfs-utils/nfs-utils.pre-install]: What created the `statd` group?

File: main/nfs-utils/APKBUILD [hunk 3]
```diff
@@ -55,6 +56,7 @@ build() {
 		--mandir=/usr/share/man \
 		--infodir=/usr/share/info \
 		--without-tcp-wrappers \
+		--with-statduser=statd \
```
Comment (tcely) [inline main/nfs-utils/APKBUILD]: Does adding `pkgusers` make sense too?

**Pull request closed without merging**